shared/output/cloud/compute_small/snippets/snippet_metadata_google.cloud.compute.v1.json

```diff
@@ -0,0 +1,575 @@
+{
+  "client_library": {
+    "name": "google-cloud-compute-v1",
+    "version": "0.0.0",
```

Comment:
Maybe just don't set it?

Reply:
It's convenient for the field to be present, because then the code that updates it on release can just regex-replace it rather than having to parse and regenerate the entire json. Is it a problem if it's present but may be set to the empty string? Or is it important for it to be absent until it can be set to an actual valid released package version?

Reply:
I think an empty string would be better than "0.0.0" as it is clearer for consumers to check for null or empty than for a specific value.